adaptors/fhir-4.md

```diff
@@ -0,0 +1,710 @@
+---
+title: FHIR r4 Adaptor
+---
+
+Language support for FHIR version 4. This adaptor is compatible with any FHIR r4
+compliant server.
+
+The FHIR 4 adaptor is implemented in TypeScript and comes complete with full
+TypeScript definitions.
+
+## About FHIR
```

Comment:
Let's link to this other docs page I have on FHIR: https://docs.openfn.org/documentation/get-started/standards#fhir-for-health-data-exchange